### PR TITLE
Use nullptr

### PR DESCRIPTION
**Pull request by mlang**

### CODE REVIEW

File: lang/LangPrimSource/PyrPrimitive.cpp [hunk 1]
```diff
@@ -2949,8 +2949,8 @@ void switchToThread(VMGlobals* g, PyrThread* newthread, int oldstate, int* numAr
         SetNil(&oldthread->block);
         SetNil(&oldthread->receiver);
         SetNil(&oldthread->frame);
-        SetRaw(&oldthread->ip, (void*)0);
-        SetRaw(&oldthread->sp, (void*)0);
+        SetRaw(&oldthread->ip, (void*)nullptr);
+        SetRaw(&oldthread->sp, (void*)nullptr);
```
Comment (brianlheim) [inline lang/LangPrimSource/PyrPrimitive.cpp]: are these  void *  casts still necessary? maybe we could remove them entirely?

**Comment by brianlheim** [comment]
also, it looks like this branch is failing the linting check in CI. could you please follow the instructions in this article (https://github.com/supercollider/supercollider/wiki/Cpp-formatting-instructions#linting-and-formatting) on how to install clang-format and use the tools/clang-format.py script to format your branch?

**Comment by mlang** [comment]
Brian Heim <notifications@github.com> writes:

> In lang/LangPrimSource/PyrPrimitive.cpp:

[...]

> are these void * casts still necessary?

Actually, yes.  They are necessary to choose the correct overload for
SetRaw.

-- 
CYa,
  ⡍⠁⠗⠊⠕

**Comment by brianlheim** [comment]
ah okay, i thought that might be the case. thanks again!

**Pull request merged**